assets/app/scripts/controllers/overview.js

```diff
@@ -289,26 +291,26 @@ angular.module('openshiftConsole')
         return;
       }
       // Make sure we have a build output
-      if (!build.parameters.output.to) {
+      if (!build.spec.output.to) {
         return;
       }
-      for (var i = 0; i < deploymentConfig.triggers.length; i++) {
-        var trigger = deploymentConfig.triggers[i];
+      for (var i = 0; i < deploymentConfig.spec.triggers.length; i++) {
+        var trigger = deploymentConfig.spec.triggers[i];
         if (trigger.type === "ImageChange") {
           var triggerImage = trigger.imageChangeParams.from.name;
```

Comment:
Hrm, the API should *never* return ImageStream to the client as a build output.  It should convert to ImageStreamTag

> On May 31, 2015, at 1:31 AM, Jordan Liggitt <notifications@github.com> wrote:
> 
> In assets/app/scripts/controllers/overview.js:
> 
> >          return;
> >        }
> > -      for (var i = 0; i < deploymentConfig.triggers.length; i++) {
> > -        var trigger = deploymentConfig.triggers[i];
> > +      for (var i = 0; i < deploymentConfig.spec.triggers.length; i++) {
> > +        var trigger = deploymentConfig.spec.triggers[i];
> >          if (trigger.type === "ImageChange") {
> >            var triggerImage = trigger.imageChangeParams.from.name;
> @bparees if a build outputs to ImageStream "foo", is that an implied output to ImageStreamTag "foo:latest"?
> 
> —
> Reply to this email directly or view it on GitHub.
>